src/states/bootstrapping.rs

```diff
@@ -264,14 +265,18 @@ impl Bootstrapping {
         Transition::Terminate
     }
 
-    fn handle_new_message(&mut self, pub_id: PublicId, bytes: Vec<u8>) -> Result<Transition> {
-        match serialisation::deserialise(&bytes) {
+    fn handle_new_message(
+        &mut self,
+        pub_id: PublicId,
+        bytes: CrustSendReceiveData,
+    ) -> Result<Transition> {
+        match self.from_curst_send_receive_data(bytes) {
```

Comment:
Typo: `from_crust_send_receive_data`
(Also: We could apply `?` here instead of matching `Err`.)